src/libraries/Common/src/System/Net/Http/Http3/Frames/Http3Frame.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+
+#if KESTREL
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3
+#else
+namespace System.Net.Http
+#endif
+{
+    internal static partial class Http3Frame
+    {
+        public static bool TryReadIntegerPair(ReadOnlySpan<byte> buffer, out long a, out long b, out int bytesRead)
```

Comment:
```suggestion
        public static bool TryReadVariableLengthIntegerPair(ReadOnlySpan<byte> buffer, out long a, out long b, out int bytesRead)
```

as the out params are longs, either we should consistently say VariableLengthInteger or Long instead of Integer.

Reply:
I feel like the current name is fine, but if others express a strong opinion I'm not opposed to changes.